roles/mattermost/files/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: "2"
+
+services:
+  db:
+    image: spaceapi/mattermost-db
+    restart: unless-stopped
+    volumes:
+      - ${DATA_FOLDER}/db/var/lib/postgresql/data:/var/lib/postgresql/data
+      - /etc/localtime:/etc/localtime:ro
+    environment:
+      - POSTGRES_USER=mmuser
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_DB=mattermost
+  app:
+    image: spaceapi/mattermost-app
+    restart: unless-stopped
+    volumes:
+      - ${DATA_FOLDER}/app/mattermost/config:/mattermost/config:rw
+      - ${DATA_FOLDER}/app/mattermost/data:/mattermost/data:rw
+      - ${DATA_FOLDER}/app/mattermost/logs:/mattermost/logs:rw
+      - /etc/localtime:/etc/localtime:ro
+    environment:
+      - MM_USERNAME=mmuser
+      - MM_PASSWORD=${POSTGRES_PASSWORD}
+      - MM_DBNAME=mattermost
+    links:
+      - db:db
+  web:
+    image: spaceapi/mattermost-web
+    ports:
+      - "8200:80"
```

Comment:
Can we document the internally used ports in the README file?

Reply:
Yep, makes it easier to see what is taken etc 
c3536a3